test/TestProjects/SqlClientApp/Program.cs

```diff
@@ -12,10 +12,19 @@ public static void Main(string[] args)
 
         public static bool Run()
         {
-            using (var client = new SqlConnection(@"Data Source=(localdb)\mssqllocaldb;Integrated Security=True"))
+            try
             {
-                client.Open();
-                return !string.IsNullOrEmpty(client.ServerVersion);
+                using (var client = new SqlConnection(@"Data Source=(localdb)\mssqllocaldb;Integrated Security=True"))
+                {
+                    client.Open();
+                    return !string.IsNullOrEmpty(client.ServerVersion);
+                }
+            }
+            catch (SqlException ex) when (ex.Number == -2)  // -2 means SQL timeout
```

Comment:
Why did the SqlException start happening with this change? Was it consistent, or just a flaky test?

Reply:
Dunno, the build pipleline failed: https://github.com/natemcmaster/DotNetCorePlugins/runs/486084609
I'm not sure if it has been a transient error.

Reply:
I haven't seen it before. This test isn't using reloading though, so it's probably not related to this change.